client/test/cypress/integration/submissiondetails.spec.js

```diff
@@ -102,9 +102,5 @@ describe("Submissions Details", () => {
       cy.wait("@addReviewer2")
     })
 
-    cy.dataCy("submission_details_notify")
-      .should("be.visible")
-      .should("have.class", "bg-negative")
-
```

Comment:
It seems this test is not asserting anything anymore?

Reply:
Added a check to confirm the assignment wasn't successful.